meetings/2021-03-15.md

```diff
@@ -0,0 +1,150 @@
+* https://lists.identity.foundation/g/sds-wg/wiki
+Matthieu: See also https://w3c-ccg.github.io/meetings/
+
+Dmitri: it is good to join the CCG mailing list as the agendas for meetings appear there.
+https://www.w3.org/community/credentials/2020/09/
+
+confidential storage group (encrypted data vaults + identity hubs) - https://identity.foundation/confidential-storage/
+(that's the spec)
+github: https://github.com/decentralized-identity/confidential-storage/
+this one: https://lists.identity.foundation/g/sds-wg/wiki
+
+
+###  [Scope discussion](https://github.com/solid/authentication-panel/issues/86)
+
+Aaron: Requesting a specific scope of "solid" seems to be more in line with what OIDC does. It would require an extra burden for the client, but it doesn't seem too cumbersome.
+
+Elf: Would it be sufficient to have a webid claim to match what is in the access token's audience claim?
+
+Aaron: When you request the scope, then the ID token and the access token would include that claim and if you don't, it wouldn't be there. It's what happens with email claims for example.
+
+Dmitri: I think requiring a scope is a good idea, it sounds like a decent way to work with the mechanism.
+
+Aaron: Can we get feedback/concerns about client tooling for example?
+
+Henry: we could put a note in the spec linking to the issue so that people can be aware of the question we are asking.
+
+Matthieu: Spec level changelog? Would it be good to have those for all the Solid specs
+
+Aaron: I've seen it in other specs and it is useful.
+
+Sarven: It's specifically useful if you have significant snapshots of the spec (WD, CR, PR, ...)
+
+Henry: Could we use the git history?
+
+Aaron: There's a lot of info in history, it will be there among people fixing less significant things.
+
+
+### Our use of the webid claim in Solid-OIDC and how we can best support DIDs
+
+Aaron: How to best support DIDs? Right now we have a webid claim and we'd like to support dids. Even though the WebID says it must be an https URL, we sort of signal it could be a DID. We could be more specific and rename the claim to solid. So that would be future proof for supporting VCs, DIDs...
+
+Dmitri: +1 to Aaron's suggestion to have a general `solid` claim that specifies both WebID and DID.
+
+Matthieu: +1 to Aaron's suggestion.
+
+Justin: That makes a lot of sense.
+
+Aaron: I spoke with Ruben V. last week and he was very supportive, he suggested making the value space for this claim IRIs.
+
+Henry: The problem remains which DID should we support? This was the question Ruben asked initially on the [Support DID issue](https://github.com/solid/specification/issues/217#issuecomment-532521655). One could argue that WebIDs could still be extended to cover DIDs. Indeed the WebID group considered for a long time covering more than `https` URLs. But we ended up adding the limitation in order to help finalize the spec. After all: DID took another 7 years to mature.
```

Comment:
Possibly relevant comments from @msporny notwithstanding, my point remains: the WebID spec should not be referred to as anything like _finalized_ because it's not! Its current shape was not even officially published as an Incubation Group (XG) Note (roughly comparable to what today would be a Community Group [CG] Report)!

Further, having been there at the time, the restriction therein to `https` URLs resulted from a flawed poll (it was issued by a single person, not the group as a whole; it was emitted through a system that bestowed improper gravity by suggesting its origination was not a single person; wording of several questions was ill considered; etc.), which was treated by some as a binding vote, treated by others as a temperature-taking straw poll, and entirely ignored by others.

Therefore --

```suggestion
Henry: The problem remains which DID should we support? This was the question Ruben asked initially on the [Support DID issue](https://github.com/solid/specification/issues/217#issuecomment-532521655). One could argue that WebIDs could still be extended to cover DIDs. Indeed the WebID group considered for a long time covering more than `https` URLs. But we ended up adding the limitation in order to help move the [WebID XG draft spec](https://www.w3.org/2005/Incubator/webid/spec/identity/) forward. After all: DID took another 7 years to mature.
```